### PR TITLE
Feature/milestone 2

### PR DESCRIPTION
**Pull request by oscardelalanza**
### What was made
- directories prepared
- App component implemented
- BookList component implemented
- Book component implemented
- Booksform component implemented
- Redux store, reducer and actions implemented

### CODE REVIEW

File: src/components/BooksList.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      <th>Book ID</th>
+      <th>Title</th>
+      <th>Category</th>
+    </tr>
+    {books.map(book => <Book key={book.id} book={book} />)}
+  </table>
+);
+const mapStateToProps = state => ({
+  books: state.books,
+});
```
Comment (Torres-ssf) [inline src/components/BooksList.jsx]: - [x] As it is requested in the instructions, we need to use the Presentational and Container Components approach. This means that components are divided based on their awareness of Redux. Container components are the ones that are aware of Redux existence and Presentation components are the ones that don't. Check the documentation [here](https://redux.js.org/basics/usage-with-react#presentational-and-container-components). Both BookList and BookForm (BookForm it's not connected to Redux yet but it's going to in a future milestone) should be placed in the `containers` directory.

File: src/components/BooksForm.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <label htmlFor="category">
+          Category:
+          <select id="category">
+            <option selected>Select Category</option>
+            {categories.map(cat => <option key={cat} value={cat}>{cat}</option>)}
+          </select>
+        </label>
+      </div>
+    </form>
+  </div>
```
Comment (Torres-ssf) [inline src/components/BooksForm.jsx]: - [x] BookForm component should be placed in the `containers` directory

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @oscardelalanza :hand: 
Nice job in this milestone so far :clap: 
But some changes are needed before moving on :muscle: 

- [x] I know your app has only one reducer right now, but let's not leave the reducer index file empty. We can make it ready to receive other reducers. Something like this:
```javascript
import { combineReducers } from 'redux'
import books from './books'

export default combineReducers({
  books,
})
```

Feel free to message me on slack if you have any questions :smile: 
Happy Coding :computer: 
**Sergio Torres**

**Comment by oscardelalanza** [comment]
### changes made
- `bookReducer` implemented in `combineReducers`
- `BooksList` and `BooksForm` moved to containers

**Review by fatymahmed — APPROVED** [review]
## Hi @oscardelalanza 
### Good job making the requested changes 🎉🎉
## Status: Approved ✔️

**Pull request merged**